nat-lab/tests/uniffi/libtelio_remote.py

```diff
@@ -188,12 +188,16 @@ def main():
     container_ip = sys.argv[2]
     port = int(sys.argv[3])
 
-    daemon = Pyro5.server.Daemon(host=container_ip, port=port)
-    wrapper = LibtelioWrapper(daemon)
-    daemon.register(wrapper, objectId=object_name)
-
-    daemon.requestLoop()
-    daemon.close()
+    try:
+        daemon = Pyro5.server.Daemon(host=container_ip, port=port)
+        wrapper = LibtelioWrapper(daemon)
+        daemon.register(wrapper, objectId=object_name)
+
+        daemon.requestLoop()
+        daemon.close()
+    except (Exception,) as e:
```

Comment:
Why not just `Exception` instead of using tuple?
Or is it to satisfy linter, because of too general exception? If yes, just disable `pylint` on that line or something :smile:

Reply:
Yeah, that was an attempt to satisfy linter, but it seems, that it didn't work :smile:  will disable that ...

Reply:
Also, now that I think. With sys.exit we kinda break flow of exception. Instead of it, just do `raise` after the print

Reply:
Fixed